bin/check_samplesheet.py

```diff
@@ -137,14 +137,24 @@ def check_samplesheet(file_in, file_out):
 
                 ## Check that replicate ids are in format 1..<num_replicates>
                 uniq_rep_ids = sorted(list(set(sample_mapping_dict[sample].keys())))
-                if len(uniq_rep_ids) != max(uniq_rep_ids):
+                if len(uniq_rep_ids) != max(uniq_rep_ids) or 1 != min(uniq_rep_ids):
                     print_error(
                         "Replicate ids must start with 1..<num_replicates>!",
                         "Sample",
                         "{}, replicate ids: {}".format(sample, ",".join([str(x) for x in uniq_rep_ids])),
                     )
                     sys.exit(1)
 
+                ## Check that multiple runs of the same sample are of the same datatype i.e. single-end / paired-end
```

Comment:
This is already on `dev` no? 
https://github.com/nf-core/atacseq/blob/e05b4fb0f0ca8b84d78802c3b413ad81d45456ef/bin/check_samplesheet.py#L149-L159

Reply:
The introductory comment line is wrong and should be something like "Check that multiple replicates of the same sample are of the same datatype". And no, this test on replicates was not in dev when I created the PR, but only libraries were tested.